test/evergreen.yml

```diff
@@ -2166,6 +2145,7 @@ tasks:
           script: |
             set -o errexit
             set -o verbose
+            sudo python3 -m pip install gcovr
```

Comment:
Can we use virtualenv here instead of messing up the system python?

Reply:
yes, I will make that change.